{
    "type": "bundle",
    "id": "bundle--2d97b58d-bb0e-4f4d-b28c-6167f4ee7fc4",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--64b7ccf2-f8d0-48f2-b7e5-670d707069bc",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-02-04T00:32:59.20705Z",
            "modified": "2022-02-04T00:32:59.20705Z",
            "name": "CVE-2022-22818",
            "description": "The {% debug %} template tag in Django 2.2 before 2.2.27, 3.2 before 3.2.12, and 4.0 before 4.0.2 does not properly encode the current context. This may lead to XSS.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-22818"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}